src/content/docs/new-relic-solutions/new-relic-one/core-concepts/new-relic-ai.mdx

```diff
@@ -19,92 +19,89 @@ import aiExplainThisErrorPopUp from 'images/ai_screenshot-crop_explain-this-erro
 
 import aiAskAiinErrorsInbox from 'images/ai_screenshot-crop_ask-ai-in-errors-inbox.webp'
 
-New Relic AI (NRAI) is designed to help every member of your team improve observability, not just engineers. We combined OpenAI's large language models (LLMs) with our unified telemetry data platform to create an AI assistant that deeply understands how both your system and New Relic function. New Relic AI helps you demystify the complexity of your stack, solve service outages, and improve your efficiency simply by letting you ask questions in plain language. 
+New Relic AI (NRAI) is designed to help every member of your team improve observability, not just engineers. We have combined OpenAI's large language models (LLMs) with our unified telemetry data platform to create an AI assistant that deeply understands how both your system and New Relic function. New Relic AI helps you demystify the complexity of your stack, solve service outages, and improve your efficiency simply by letting you ask questions in plain language. 
+
+Ask New Relic to help you:
+* [Analyze error logs](https://docs.newrelic.com/docs/new-relic-solutions/new-relic-one/core-concepts/new-relic-ai/#errors)
+* [Dive deeper in stack traces](https://docs.newrelic.com/docs/new-relic-solutions/new-relic-one/core-concepts/new-relic-ai/#stack-traces)
+* [Identify gaps in your alerting ecosystem](https://docs.newrelic.com/docs/new-relic-solutions/new-relic-one/core-concepts/new-relic-ai/#coverage-gaps)
+* [Create synthetic monitors](https://docs.newrelic.com/docs/new-relic-solutions/new-relic-one/core-concepts/new-relic-ai/#)
+* [Understand your dashboards](https://docs.newrelic.com/docs/new-relic-solutions/new-relic-one/core-concepts/new-relic-ai/#dashboards)
+* [Check the status of support tickets](https://docs.newrelic.com/docs/new-relic-solutions/new-relic-one/core-concepts/new-relic-ai/#support)
+
+<Video
+ type="youtube"
+ id="7vBhF4Ph_LA?si=UyaTG9LwSfkqVUU5"
+/>
 
 ## Ways to interact with New Relic AI [#work-with-NRAI]
 
-We help you get to the heart of your system by offering quick and easy ways to ask questions: AI chat, and the NRQL console. 
+We help you get to the heart of your system by offering quick and easy ways to ask questions. You can chat with our AI and interact with it in the NRQL console or work with some of our AI tools directly in the UI.
```

Comment:
"you can chat and interact with our AI" helps shorten the sentence I think

Reply:
Also, there should be a comma between 'console' and 'or', I THINK.

Reply:
fixed